superbench/cli/_handler.py

```diff
@@ -21,49 +21,104 @@ def check_argument_file(name, file):
         name (str): argument name.
         file (str): file path.
 
+    Returns:
+        str: Absolute file path if it exists.
+
     Raises:
         CLIError: If file does not exist.
     """
-    if file and not Path(file).exists():
-        raise CLIError('{} {} does not exist.'.format(name, file))
+    if file:
+        if not Path(file).exists():
+            raise CLIError('{} {} does not exist.'.format(name, file))
+        return str(Path(file).resolve())
+    return file
 
 
-def version_command_handler():
-    """Print the current SuperBench tool version.
+def split_docker_domain(name):
+    """Split Docker image name to domain and remainder part.
+
+    Ported from https://github.com/distribution/distribution/blob/v2.7.1/reference/normalize.go#L62-L76.
+
+    Args:
+        name (str): Docker image name.
 
     Returns:
-        str: current SuperBench tool version.
+        str: Docker registry domain.
+        str: Remainder part.
     """
-    return superbench.__version__
+    i = name.index('/')
```

Comment:
index will raises an exception if the value is not found. Shall we use find? https://www.w3schools.com/python/ref_string_index.asp

Reply:
sure